ddl/util/util.go

```diff
@@ -172,6 +172,11 @@ func UpdateDeleteRange(sctx sessionctx.Context, dr DelRangeTask, newStartKey, ol
 	return errors.Trace(err)
 }
 
+// LoadDDLDistributeVars loads ddl distribute reorg variable from mysql.global_variables.
+func LoadDDLDistributeVars(ctx context.Context, sctx sessionctx.Context) error {
+	return LoadGlobalVars(ctx, sctx, []string{variable.TiDBDDLEnableDistributeReorg})
+}
+
```

Comment:
Perhaps we don't need this because of the new global variable synchronization mechanism.